src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -39,7 +40,7 @@ public void Direct_Producer_validation(string inputFile)
             string instanceText = File.ReadAllText(inputFile);
             var validator = new Validator(_schema);
 
-            string[] errors = validator.Validate(instanceText);
+            Result[] errors = validator.Validate(instanceText, inputFile);
```

Comment:
And here's the whole point; the validator returns `Result` objects. It also takes the name of the input file so it can include those in the `Result` objects.